source/includes/_invoice-payment.md

```diff
@@ -1449,9 +1072,6 @@ class Tag {
    }
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?